packages/jumpstarter-driver-flashers/oci_bundles/aarch64-itb/replace_kernel.sh

```
@@ -15,8 +15,8 @@ KMOD=(
 
 [ $# -ne 1 ] && { echo "$0 [target_overlay_dir]"; exit 1; }
 ODIR=$1
-mkdir -p /kernel-automotive
-pushd /kernel-automotive
+mkdir -p ./kernel-automotive
+pushd ./kernel-automotive
```

Shellcheck (0.10.0)

[warning] Use 'pushd ... || exit' or 'pushd ... || return' in case pushd fails.

(SC2164)